tests/foreman/cli/test_contentviews.py

```diff
@@ -250,6 +250,94 @@ def test_cv_delete(self):
         with self.assertRaises(CLIReturnCodeError):
             ContentView.info({'id': con_view['id']})
 
+    def test_delete_cv_files(self):
+        """@Test: Delete content view and verify it was actually deleted from
```

Comment:
to make the test description more clear: 
s/Delete content view/Delete empty content view